content/challenges/hard/3 - Missing Number/func.js

```diff
@@ -10,5 +10,9 @@
  * @returns {Number} the missing number
  */
 module.exports = (arr) => {
-    // Your code here
+    let expectedNumbers = [];
+    for (i = 1; i <= arr.length + 1; i++) {
+        expectedNumbers.push(i)
+    }
+    return expectedNumbers.filter(n => !arr.includes(n))[0]
```

Comment:
This works! Try to rework this using math

Hint: if n=4 and your array is [1,3,4] how might you use math to find out that the 2 is what is missing?